Resources/Prototypes/Datasets/figurines.yml

```
@@ -0,0 +1,263 @@
+- type: localizedDataset
+  id: FigurinesHoP
+  values:
+    prefix: figurines-hop-
+    count: 2
+
+- type: localizedDataset
+  id: FigurinesPassenger
+  values:
+    prefix: figurines-passenger-
+    count: 2
+
+- type: localizedDataset
+  id: FigurinesGreytider
+  values:
+    prefix: figurines-greytider-
+    count: 5
+
+- type: localizedDataset
+  id: FigurinesClown
+  values:
+    prefix: figurines-clown-
+    count: 5
+
+- type: localizedDataset
+  id: FigurinesHoloClown
+  values:
+    prefix: figurines-holoclown-
+    count: 1
+
+- type: localizedDataset
+  id: FigurinesMime
+  values:
+    prefix: figurines-mime-
+    count: 5
+
+- type: localizedDataset
+  id: FigurinesMusician
+  values:
+    prefix: figurines-musician-
+    count: 2
+
+# - type: localizedDataset # TODO add something
+#   id: FigurinesBoxer
+#   values:
+#     prefix: figurines-boxer-
+#     count: 0
+
+- type: localizedDataset
+  id: FigurinesCaptain
+  values:
+    prefix: figurines-captain-
+    count: 2
+
+- type: localizedDataset
+  id: FigurinesHoS
+  values:
+    prefix: figurines-hos-
+    count: 2
+
+- type: localizedDataset
+  id: FigurinesWarden
+  values:
+    prefix: figurines-warden-
+    count: 1
+
+- type: localizedDataset
+  id: FigurinesDetective
+  values:
+    prefix: figurines-detective-
+    count: 1
+
+- type: localizedDataset
+  id: FigurinesSecurity
+  values:
+    prefix: figurines-security-
+    count: 2
+
+- type: localizedDataset
+  id: FigurinesLawyer
+  values:
+    prefix: figurines-lawyer-
+    count: 2
+
+- type: localizedDataset
+  id: FigurinesCargoTech
+  values:
+    prefix: figurines-cargotech-
+    count: 3
+
+- type: localizedDataset
+  id: FigurinesSalvage
+  values:
+    prefix: figurines-salvage-
+    count: 1
+
+- type: localizedDataset
+  id: FigurinesQM
+  values:
+    prefix: figurines-qm-
+    count: 7
+
+- type: localizedDataset
+  id: FigurinesCE
+  values:
+    prefix: figurines-ce-
+    count: 6
```

```suggestion
    count: 5
```
Should fix the integration test failing

---

oh right, i forgor